photutils/psf/__init__.py

```diff
@@ -7,3 +7,5 @@
 from .funcs import *
 from .models import *
 from .groupstars import *
+from .psfphotometry import *
```

Comment:
maybe rename `psfphotometry` to just `photometry`?  after all, it's in `psf` already, so `psf.photometry` makes more sense than `psf.psfphotometry`.

(both the import here and the file name)